src/rpc/mining.cpp

```diff
@@ -106,13 +106,14 @@ static UniValue generatetoaddress(const JSONRPCRequest& request)
 
     int nGenerate = request.params[0].get_int();
 
-    CTxDestination destination = DecodeDestination(request.params[1].get_str());
+    ColorIdentifier* colorId = nullptr;
+    CTxDestination destination = DecodeDestination(request.params[1].get_str(), colorId);
```

Comment:
Since the following definitions are in `key_io.h`, is it okay to keep the original code?

```cpp
CTxDestination DecodeDestination(const std::string& str, ColorIdentifier* colorId = nullptr);
```

other code also same.

Reply:
actually the colorId will be use at line 116 GetScriptForDestination function
`coinbaseScript->reserveScript = GetScriptForDestination(destination, colorId);`

Reply:
Oh, Isee.